alexandrie/src/utils/request_log.rs

```diff
@@ -22,7 +22,17 @@ impl<State: Clone + Send + Sync + 'static> Middleware<State> for RequestLogger {
         let res = next.run(req).await;
         let elapsed = start.elapsed().as_millis();
         let status = res.status();
-        info!("--> {} {} {} {}ms", method, path, status, elapsed,);
+        let mut resp_msg = {
```

Comment:
We can drop the `mut` keyword here, since we're not mutating it anymore.
```suggestion
        let resp_msg = {
```

Reply:
You are right.